src/lib/tooltip/Tooltip.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+	import { createTooltip } from '@melt-ui/svelte';
+	import { fade } from 'svelte/transition';
+	import { twMerge } from 'tailwind-merge';
+
+	export let placement = 'top';
+	export let tooltipClass = '';
```

Comment:
lets default this to undefined rather than (ab)using the empty string for this state